Vagrantfile

```diff
@@ -27,10 +27,13 @@ $shared_folders = {}
 $forwarded_ports = {}
 $subnet = "172.17.8"
 $os = "ubuntu"
+$network_plugin = "flannel"
 # The first three nodes are etcd servers
 $etcd_instances = $num_instances
-# The first two nodes are masters
+# The first two nodes are kubernetes masters
```

Comment:
can you make it consistent? kube master and kube node?

Reply:
sure

Reply:
fixed with amend commit :)